src/java/com/unifina/signalpath/blockchain/HttpEthereumJsonRpc.java

```diff
@@ -14,6 +14,7 @@
 class HttpEthereumJsonRpc extends EthereumJsonRpc{
 	HttpEthereumJsonRpc(String url, JsonRpcResponseHandler handler) {
 		super(url, handler);
+		handler.init();
```

Comment:
in this instance handler.init() installs filter

Reply:
Here too it would be better to comment in code instead of here :)